opcodes.js

```diff
@@ -135,21 +135,21 @@ const codes = {
   0x9e: ['SWAP', 3, 0, 0],
   0x9f: ['SWAP', 3, 0, 0],
 
-  0xa0: ['LOG', 0, 2, 0],
-  0xa1: ['LOG', 0, 3, 0],
-  0xa2: ['LOG', 0, 4, 0],
-  0xa3: ['LOG', 0, 5, 0],
-  0xa4: ['LOG', 0, 6, 0],
+  0xa0: ['LOG', 375, 2, 0],
+  0xa1: ['LOG', 375, 3, 0],
+  0xa2: ['LOG', 375, 4, 0],
+  0xa3: ['LOG', 375, 5, 0],
+  0xa4: ['LOG', 375, 6, 0],
```

Comment:
I wonder if it is properly charging for the data too?

Reply:
Can somebody check this?

Reply:
As discussed just now, `evm2wasm` does not charge for the data